docker-compose.ci.yml

```diff
@@ -1,86 +1,35 @@
 version: '3'
 
-
 services:
-    web:
-        image: nginx:1.13.7
-        ports:
-            # TODO: must be configurable/random
-            - "8080:80"
-        volumes:
-            - ./config/nginx-default.conf:/etc/nginx/conf.d/default.conf
-            # TODO: can we avoid this by removing try_files?
-            - ./web:/srv/annotations/web
-        depends_on:
-            - fpm
-    fpm:
-        build: 
+    composer:
+        build:
+            args:
+                composer_dev: --no-dev
+    composer-dev:
```

Comment:
I'm not sure how far `docker-compose.yml` and `docker-compose.ci.yml` are from unification right now, but maybe we should put both these `composer` images in `docker-compose.yml` to have a complete stack there. If I want to run tests locally I shouldn't need to use `docker-compose.ci.yml`?

Reply:
Separate them to avoid a double-Composer build in dev. PHPUnit works using `cli`, but `project_tests.sh` won't because of Proofreader.